src/app/cases/styles.tsx

```diff
@@ -16,7 +16,7 @@ export const FiltersContainer = styled.div`
 
 export const MainDisplay = styled.main`
   display: grid;
-  grid-template-columns: 5fr 10fr;
+  grid-template-columns: 4fr 10fr;
   margin-top: 2rem;
   width: 100%;
 `;
```

Comment:
would it be possible for us to change things to just have the column of case listing cards be scrollable? especially now that the interest form seems like it will generally be scrollable....i think having both of them being independently scrollable is the move, as opposed to what it currently is which is the whole page being scrollable and having that scroll through the listing cards?

@varortz i know you mentioned changing this in an interest form PR but lowk it might be ideal to add it in here?

(cc @jinkang-0 @davidqing6432)

Reply:
yes agreed, we may want a redesign to account for the scrollbars though @davidqing6432 (perhaps even a styled scrollbar given how ugly it is on windows)

Reply:
fixed the scrolling 🥰